### ISSUE TITLE
Lively Wallpaper very slow to apply wallpaper after restart in Windows 11 24H2

### ISSUE THREAD

**Issue by Ryanthelion09** [rocksdanister/lively#2422]
Hey everyone! This is my first time posting on here, so let me know if I'm omitting any information or forgetting to do something :)
**Describe the bug**
I recently updated to Windows 11 version 24H2, but now whenever I restart my computer, Lively is very slow to apply my wallpaper. The Lively App is run and is minimized to my system tray (as usual), but it just doesn't apply my wallpaper. Even if I go to the system tray, click "Open Lively", and then click my desired wallpaper, nothing happens. The only way (that I've found) to get my wallpaper applied is to wait 3-5 minutes, and then magically it starts working as intended. This issue did not exist when I was on Windows 11 version 23H2.

**To Reproduce**
1. Have Windows 11 v24H2 installed
2. Restart PC

**Expected behavior**
Upon restarting, the wallpaper that was displayed prior to the restart should be applied within 15 seconds.

**Screenshots/Video**
If applicable, add screenshots/video to help explain your problem.

**Desktop (please complete the following information):**
 - OS: Windows 11 24H2
 - Which wallpaper was running at the time if any: Rain

**Additional context**

**Log file (Important)**
Share the diagnostic files by:
Open Lively settings (Gear icon) top-right -> System -> Log File -> 
[lively_log_20241006_192925.zip](https://github.com/user-attachments/files/17272008/lively_log_20241006_192925.zip)
Save
or 
Navigate to Appdata folder `C:\Users\<UserName>\AppData\Local\Lively Wallpaper\logs`

**Comment by rocksdanister** [comment]
Some users have reported turning HDR on and off fixed the issue: Windows + Alt + B.

I haven't received 24H2 update yet, waiting for it.

**Comment by Ryanthelion09** [comment]
I already have HDR turned off. It appears that Microsoft is aware of this issue: https://learn.microsoft.com/en-us/windows/release-health/status-windows-11-24h2#3370msgdesc

**Comment by TheQuickFox** [comment]
> Some users have reported turning HDR on and off fixed the issue: Windows + Alt + B.
> 
> I haven't received 24H2 update yet, waiting for it.

If you don't want to wait or need a copy of 24H2 for testing: You can 24H2 now with the media creation tool and do an upgrade from your current Windows 11 install:
https://www.microsoft.com/en-us/software-download/windows11

If you have incompatible hardware use this script before running setup:
https://github.com/AveYo/MediaCreationTool.bat/blob/main/bypass11/Skip_TPM_Check_on_Dynamic_Update.cmd

**Comment by Ryanthelion09** [comment]
Yes, or you could join Windows Insiders and choose the Release Preview channel to get access to the update.
